projects/kit/utils/math/test/key-steps.spec.ts

```diff
@@ -35,7 +34,6 @@ describe(`KeySteps utils`, () => {
 
     describe(`tuiKeyStepValueToPercentage`, () => {
         for (const {value, percentage} of testContexts) {
-            // eslint-disable-next-line no-loop-func
```

Comment:
Will it solves error if the change this line to the following way ?

```ts
describe(`utility tuiKeyStepValueToPercentage`, () => {
```

Reply:
`no-loop-func` doesn't respect now